code/datums/status_effects/buffs.dm

```diff
@@ -173,8 +174,10 @@
 		return FALSE
 	ADD_TRAIT(owner, TRAIT_CHUNKYFINGERS, VAMPIRE_TRAIT)
 	var/mob/living/carbon/human/H = owner
-	H.physiology.brute_mod *= 0.5
-	H.physiology.burn_mod *= 0.8
+	original_color = H.color
+	H.color = "#a61915"
```

Comment:
not code review but I really dont like the colouration. I dont know it just looks- too much.

Reply:
Removed - apparently matrix recolouration just sucks and I trust AA and Charlie on this